README.md

```diff
@@ -9,13 +9,20 @@ npm run serve
 ```
 
 ### Customize configuration
+
 See [Configuration Reference](https://cli.vuejs.org/config/).
 
-## Step 1 : Avoir le temps du jour pour Grenoble
+## Step 1 : Afficher la météo du jour pour Grenoble
+
+- Sur la page d'une ville récupérer et afficher la météo du jour.
+- Le page d'une ville correspond au composant `City` 
+- Utiliser le service d'API pour récupérer la météo en function des coordonnées GPS de Grenoble lorsque le composant est créé (https://vuejs.org/v2/guide/instance.html#Instance-Lifecycle-Hooks)
+- Binder les données de météo avec le template (https://vuejs.org/v2/guide/instance.html#Data-and-Methods)
 
-Sur la page d'une ville récupérer e afficher le temps du jour.
+## Step 2 : Afficher la météo du jour pour toutes les villes Zenika
 
-## Step 2 : Avoir le temps du jour pour toutes les villes Zenika
+- Utiliser les getters `Vuex` pour récupérer les coordonnées de la ville récupérée des `props` du composant (https://vuex.vuejs.org/guide/getters.html)
```

Comment:
Je pensais qu'on voulait faire afficher la liste des villes dans cette seconde étape.
Quitte à écrire vraiment en dur la première ville dans `Home.vue`. 

Ça permettrait de laisser les gens récupérer la liste des villes ici et les faire utiliser un `v-for` et le paramétrage de la route.

Reply:
Oui je crois bien que t'as raison, j'avais pas vu que le composant `Home` n'affiche pas toutes les villes